### PR TITLE
Refactor turn.rest API to match feedback

### PR DESCRIPTION
**Pull request by mgherghe**

### CODE REVIEW

File: service/turn.rest/src/test/java/org/kaazing/gateway/service/turn/rest/TurnRestIT.java [hunk 1]
```diff
@@ -67,18 +67,14 @@
...
-                                .property("uri", "uri3")
-                            .done()
-                            .nestedProperty("options")
-                                .property("credentials.ttl", "43200")
-                                .property("username.separator", ":")
-                            .done()
+                            .property("credentials.generator", "class:" + DefaultCredentialsGenerator.class.getName())
+                            .property("credentials.ttl", "43200")
+                            .property("username.separator", ":")
+                            .property("url", "url1")
```
Comment (dpwspoon) [inline service/turn.rest/src/test/java/org/kaazing/gateway/service/turn/rest/TurnRestIT.java]: please change the url1, url2, url3 to be real examples:

for example
"turn:1.2.3.4:9991?transport=udp",
       "turn:1.2.3.4:9992?transport=tcp",
       "turns:1.2.3.4:443?transport=tcp"

as from Spec https://tools.ietf.org/html/draft-uberti-rtcweb-turn-rest-00

File: service/turn.rest/src/main/java/org/kaazing/gateway/service/turn/rest/internal/TurnRestService.java [hunk 2]
```diff
@@ -67,8 +69,7 @@ public void init(ServiceContext serviceContext) throws Exception {
         String uris = getTurnURIs(properties);
```
Comment (dpwspoon) [inline service/turn.rest/src/main/java/org/kaazing/gateway/service/turn/rest/internal/TurnRestService.java]: consider changing %s/uri/url/g

**Comment by dpwspoon** [comment]
LGTM, 2 minor comments

File: server/src/main/xsd/gateway-config-201616.xsd [hunk 3]
```diff
@@ -1622,44 +1622,32 @@
...
-                        <complexType>
-                            <sequence>
-                                <element maxOccurs="unbounded" minOccurs="1" name="uri"
-                                    type="gateway:CollapsedString">
-                                    <annotation />
-                                </element>
-                            </sequence>
-                        </complexType>
+                    <element maxOccurs="1" minOccurs="0" name="credentials.ttl"
+                        type="gateway:CollapsedString">
```
Comment (dpwspoon) [inline server/src/main/xsd/gateway-config-201616.xsd]: This should not be of typed collapsed gateway:TimeIntervalString, I believe

**Comment by mgherghe** [comment]
Addressed feedback comments.

**Pull request merged**